lib/src/widgets/links_card_widget/links_card_widget.dart

```diff
@@ -30,18 +45,15 @@ class LinksCardWidget extends StatelessWidget {
           children: options,
         ),
         Padding(
-          padding: const EdgeInsets.only(
-            top: 1,
-            bottom: 10,
-            left: 3,
-            right: 3,
-          ),
-          child: Container(
-            color: const Color(0xffe5e5e5),
-            height: 1,
+          padding: dividerPadding,
+          child: const Divider(
+            height: 0,
```

Comment:
Why the zero-divier here?

Reply:
Because height in Divider makes same spacing for top and bottom. But we need differently values and i used Padding for this